lib/lintPostcssResult.mjs

```diff
@@ -3,16 +3,17 @@ import { EOL } from 'node:os';
 import { DEFAULT_SEVERITY, RULE_NAME_ALL } from './constants.mjs';
 import { DEFAULT_CONFIGURATION_COMMENT } from './utils/configurationComment.mjs';
 import assignDisabledRanges from './assignDisabledRanges.mjs';
+import emitDeprecationWarning from './utils/emitDeprecationWarning.mjs';
 import getStylelintRule from './utils/getStylelintRule.mjs';
 import reportUnknownRuleNames from './reportUnknownRuleNames.mjs';
 import rules from './rules/index.mjs';
 
-/** @import {Config as StylelintConfig, LinterOptions, PostcssResult} from 'stylelint' */
+/** @import {Config, LinterOptions, PostcssResult, RuleContext} from 'stylelint' */
```

Comment:
[suggest] `RuleContext` is no longer used, right?

```suggestion
/** @import {Config, LinterOptions, PostcssResult} from 'stylelint' */
```

Btw, `eslint` or `tsc` don't seem to detect such an unused import. 😅

Reply:
eslint-plugin-jsdoc#1165

Reply:
@ybiquitous should we add https://github.com/gajus/eslint-plugin-jsdoc/blob/main/docs/rules/no-undefined-types.md to `eslint-config-stylelint`?